### PR TITLE
Add LatinX News

### PR DESCRIPTION
**Pull request by andres9403**
Adding news about the LatinXinChE award

### CODE REVIEW

File: _posts/2023-09-27 LatinX.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: LatinXinChE First Virtual Symposium 2023
+description: 'Speaker Session Winners!'
+image: assets/images/posts/LatinX_Final.png
+---
+
+We are pleased to announce that our both visiting scholars [Andres](https://SECQUOIA.github.io/2-members.html) and [Pedro](https://SECQUOIA.github.io/2-members.html) have been awarded in the **1st [LatinXinChE](https://twitter.com/LatinXinChE) Virtual Symposium** in the speaking session.
```
Comment (bernalde) [inline _posts/2023-09-27 LatinX.md:8]:
```suggestion
We are pleased to announce that both our visiting scholars [Andres](https://SECQUOIA.github.io/2-members.html) and [Pedro](https://SECQUOIA.github.io/2-members.html) have received awards in the **1st [LatinXinChE Virtual Symposium](https://www.linkedin.com/posts/latinxinche_hispanicheritagemonth-latinxinche-activity-7110006993557323776-3gb9)**.
Pedro and Andres were the only finalists who were not faculty or Ph.D. candidates. Yet, Andres got the highest award, allowing him to talk at AIChE and fund his visit to Orlando. Pedro got a memorable mention and a monetary prize!
```

File: _posts/2023-09-27 LatinX.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+<div style="text-align: center"> <img style='height: 50%; width: 80%' src="{% link assets/images/posts/Added_Twitter_LatinX.png %}" alt=""/> </div>
+
+<ul class="actions">
+    <li><a href="/3-news.html" class="button icon fa-arrow-left">Back</a></li>
+</ul>
+
+More than 50 scientist from all around the world 🌐 shared their cutting-edge researches. We want to thank you [LatinXinChE](https://twitter.com/LatinXinChE) for this great opportunity to share our work and to meet other LatinX scientists. [#LatinXinChESymposium2023](https://twitter.com/hashtag/LatinXinChESymposium2023?src=hashtag_click)
```
Comment (bernalde) [inline _posts/2023-09-27 LatinX.md:16]:
```suggestion
More than 50 scientists from all around the world 🌐 shared their cutting-edge research. Thank you [LatinXinChE](https://www.aiche.org/community/sites/committees/minority-affairs/latinx-che) for this excellent opportunity to share our work!
```

**Review by bernalde — CHANGES_REQUESTED** [review]
Small changes! Will post on LinkedIn and @ZedongPeng on the Engineering website!

**Comment by ZedongPeng** [comment]
No problem. I will update the Engineering website.

**Review by bernalde — APPROVED** [review]

**Comment by ZedongPeng** [comment]
The post doesn't appear on the group website. Try to replace the blank in the file name.  @andres9403 
`2023-09-27 LatinX.md` to `2023-09-27-LatinX.md`.

**Comment by ZedongPeng** [comment]
The engineering website updated.

**Pull request merged**